common/remote/sockfile.cpp

```diff
@@ -470,6 +470,36 @@ static const char *getRFSERRText(unsigned err)
     return "RFSERR_Unknown";
 }
 
+unsigned mapDafilesrvixCodes(unsigned err)
+{
+    switch (err)
+    {
+        case 8200:
+            return RFSERR_InvalidCommand;
+        case 8201:
+            return RFSERR_NullFileIOHandle;
+        case 8202:
+            return RFSERR_InvalidFileIOHandle;
+        case 8203:
+            return RFSERR_TimeoutFileIOHandle;
+        case 8204:
+            return RFSERR_OpenFailed;
+        case 8205:
+            return RFSERR_ReadFailed;
+        case 8206:
+            return RFSERR_WriteFailed;
+        case 8207:
+            return RFSERR_RenameFailed;
+        case 8208:
+            return RFSERR_SetReadOnlyFailed;
+        case 8209:
+            return RFSERR_GetDirFailed;
+        case 8210:
+            return RFSERR_MoveFailed;
```

Comment:
It would worth adding a comment at top of this method and point to daliservix.cpp and the definitions of this constants there.

Reply:
Done